coalib/misc/Exceptions.py

```diff
@@ -20,10 +15,10 @@ def get_exitcode(exception, log_printer=None):
         exitcode = exception.code
     elif isinstance(exception, VersionConflict):
         log_message = Constants.VERSION_CONFLICT_MESSAGE % str(exception.req)
-        log_printer.log_exception(log_message, exception)
+        LogPrinterMixin().log_exception(log_message, exception)
```

Comment:
Right now, most of the logs of type `DEBUG` are generated on `stdout`. I've mainly analysed logs like `Running bear ...`, and it is coming in `stdout`. Due to which couple of tests are failing.

I've tried changing logging config using `logging.basicConfig(stream=sys.stderr, log_level=logging.DEBUG)`, but it didn't help.

Let me know if you've any clue about that.